HaskellDates.hsproj/HaskellDates.hs

```diff
@@ -0,0 +1,62 @@
+import Data.Time
+import Data.Time.Calendar.OrdinalDate
+import Control.Applicative
+
+data Event = Event {
+  date :: Day,
+  daysOfWeek :: [Int],
+  week :: Int
+}
+
+-- WeekDay for debugging purposes
+newtype WeekDay = WeekDay Int
```

Comment:
it might be nice to just ditch the type and have a function that will just print a word for the number